.github/actions/publish-coverage/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Container with dependencies necessary to run unit-tests and code coverage
+
+# Build development environment
+# $ docker build . --tag note-arduino-coverage
+
+# Launch development environment (mount source root as /note-arduino/)
+# $ docker run --rm --volume $(pwd)/../../../:/note-arduino/ --workdir /note-arduino/ note-arduino-coverage
+
+
+# Base Image - here we reuse the test image since it has all the dependencies required 
+FROM frolvlad/alpine-python3
```

Comment:
Do we trust `frolvlad`?

Reply:
I'll use the alpine base image and install python.